src/graphql/resolvers/ssb.js

```diff
@@ -42,6 +123,110 @@ module.exports = function Resolvers () {
     })
   }
 
+  /**
+   * Gets all the votes on a message
+   * TODO: doesnt handle unlikes and duplicates yet.
+   * @param {string} id - id of a message
+   */
+  const getVotes = (id) => {
+    return new Promise((resolve, reject) => {
+      pull(
+        ssb.db.query(
+          where(
+            votesFor(id)
+          ),
+          descending(), // latest => oldest
```

Comment:
:fire:
remove this `descending` as not needed. Mainly important for when we want to reduce the like/unlike stuff - this will need to be applied "in order"

This particular resolver would be cool to write a test over:
- Like
- Like + Like
- Like + Unlike
- Unlike

etc.